projects/media-viewer/src/lib/viewers/image-viewer/image-viewer.component.html

```diff
@@ -1,11 +1,18 @@
 <div id="viewer-wrapper">
-  <ng-container *ngFor="let annotation of annotationSet?.annotations">
-    <mv-annotation [annotation]="annotation" [zoom]="zoom"
-                   [selectedAnnotation]="selectedAnnotation"
-                   (update)="updateAnnotation($event)"></mv-annotation>
-  </ng-container>
-  <div *ngIf="!errorMessage" id="image-container" class="image-container" [ngStyle]="{ transform: zoomStyle }">
-    <img #img *ngIf="url" data-hook="dm.viewer.img" [src]="url" [ngStyle]="{ transform: rotationStyle }" (error)="onLoadError()"/>
+  <div *ngIf="!errorMessage" id="image-container" class="image-container" [ngClass]="'rot' + rotation">
+
+    <img #img
+         *ngIf="url"
+         data-hook="dm.viewer.img"
+         [src]="url"
+         (error)="onLoadError()"
+         [style.width]="img.naturalWidth * zoom + 'px'"/>
+
+    <mv-annotations [annotations]="annotationSet?.annotations"
+                    [zoom]="zoom"
+                    [selectedAnnotation]="selectedAnnotation">
```

Comment:
this is not needed anymore, as we're just listening to the focusOut event on the annotation itself